pages/data-design/collaboration/members.mdx

```diff
@@ -6,7 +6,27 @@ Expand your Avo team and manage your members access by opening the workspace set
 
 ## Roles
 
-You can assign roles when inviting new users as well as for existing users, excluding yourself.
+Admins can assign roles when inviting new users as well as for existing users, excluding yourself.
+
+Role overview:
+
+|                                        | [**Billing**](/data-design/collaboration/members#billing-only) | [**Viewer**](/data-design/collaboration/members#viewer) | [**Editor**](/data-design/collaboration/members#editor) | [**Admin**](/data-design/collaboration/members#admin) |
```

Comment:
Can we un-bold these header items for a more balanced look?

Reply:
yes we can!